sda-commons-dependencies/build.gradle

```diff
@@ -88,7 +81,12 @@ dependencies {
     api "org.flywaydb:flyway-core:6.5.0"
 
     // sda-commons-server-hibernate-testing
-    api "com.github.database-rider:rider-core:1.2.9" // latest version that supports Parameterized tests: https://github.com/database-rider/database-rider/issues/104
+    api "com.github.database-rider:rider-core:1.2.9", {
+      because "latest version that supports Parameterized tests: https://github.com/database-rider/database-rider/issues/104"
+    }
+    api "org.dbunit:dbunit:2.7.0", {
```

Comment:
Does it include security updates? If not I am not sure if this version works well with rider-core (https://github.com/database-rider/database-rider/issues/111#issuecomment-452425777).

Reply:
I only checked to provide the same versions as in [master](https://github.com/SDA-SE/sda-dropwizard-commons/blob/master/sda-commons-dependencies/build.gradle#L110).

Reply:
But that doesn't necessarily mean it made sense before. But I'm fine with leaving it on 2.7.